inject-kotlin-test/src/main/kotlin/io/micronaut/annotation/processing/test/support/AbstractKotlinCompilation.kt

```diff
@@ -293,7 +293,6 @@ abstract class AbstractKotlinCompilation<A : CommonCompilerArguments> internal c
 
 internal fun convertKotlinExitCode(code: ExitCode) = when(code) {
     ExitCode.OK -> KotlinCompilation.ExitCode.OK
-    ExitCode.OOM_ERROR,
```

Comment:
why are we removing this?

Reply:
As per [the file comment](https://github.com/micronaut-projects/micronaut-core/pull/9793#discussion_r1310094295), we had duplicate labels in the when (we had OOM_ERROR twice)

I removed this first one, so it will throw an OOM error instead of a generic one